dbt/include/snowflake/macros/catalog.sql

```diff
@@ -37,6 +37,8 @@
 
 
 {% macro snowflake__get_catalog_tables_sql(information_schema) -%}
+    {%- set enable_stats = env_var('DBT_CATALOG_TABLE_ENABLE_STATS', 'true') == 'true' -%}
```

Comment:
Is there precedent for using an env_var this way in macros? I suspect the Adapters team might want us to have an official flag for this.